### ISSUE TITLE
Can't define my own template

### ISSUE THREAD

**Issue by nedbat** [manicmaniac/betterga#1]
In my .vimrc I have:
```
silent! call plug#begin()
"... lots of other plugins ...
Plug 'manicmaniac/betterga'
let g:betterga_template = '<{ci.char}> [{ci.name} ({ci.category})] {ci.ord}, {ci.hex}, o{ci.oct}'

call plug#end()
```
But when I use `ga`, I still have the default global template.

**Comment by manicmaniac** [comment]
@nedbat Thank you for your report.

I fixed this (https://github.com/manicmaniac/betterga/pull/2) and released as 0.3.2.
Please try updating the plugin.

**Comment by nedbat** [comment]
Works great, thanks! :)
